suite2p/run_s2p.py

```diff
@@ -59,6 +59,7 @@ def default_ops():
         'save_NWB': False,  # whether to save output as NWB file
         'combined': True,  # combine multiple planes into a single result /single canvas for GUI
         'aspect': 1.0,  # um/pixels in X / um/pixels in Y (for correct aspect ratio in GUI)
+        'report_time': True,  # whether to return a plane-specific dictionary with times for each pipeline stage
```

Comment:
If it defaults to true and doesn't really have much of a cost to implement, then maybe it's not necessary to include it as a feature flag in the ops--it could just always be on.

Reply:
Good point, I'll make this change